topology/probes/runc/etc-hosts

```diff
@@ -0,0 +1,8 @@
+# Kubernetes-managed hosts file.
```

Comment:
1/ don't include this file here for the test only
2/ insert space,tab in front of the IPs and comment
3/ add comment at the end of the line like   ip host #comment

Reply:
1/ - `readHosts` accepts a file anyway
2/ - handled